packages/beacon-node/src/util/queue/options.ts

```diff
@@ -16,6 +16,8 @@ export type JobQueueOpts = {
   type?: QueueType;
   /**By default yield per push()*/
   noYieldIfOneItem?: boolean;
+  /** If set will not raise abort error instead drop all jobs */
+  dropAllJobOnAbort?: boolean;
```

Comment:
I think this will bury errors.  If there is an issue processing the queue it should be raised.  Silently dropping jobs does not seem like the right path forward but im open to discuss this further